db/install.xml

```diff
@@ -36,8 +36,8 @@
                 <FIELD NAME="nondisclosure" TYPE="int" LENGTH="1" NOTNULL="true" UNSIGNED="true" DEFAULT="0" SEQUENCE="false" COMMENT="Flag to mark nondisclosure documents." PREVIOUS="username" NEXT="show_report_to_students" />
                 <FIELD NAME="show_report_to_students" TYPE="int" LENGTH="1" NOTNULL="true" UNSIGNED="true" DEFAULT="0" SEQUENCE="false" COMMENT="Flag to show the report to students" PREVIOUS="nondisclosure"/>
                 <FIELD NAME="show_students_links" TYPE="int" LENGTH="1" NOTNULL="true" UNSIGNED="true" DEFAULT="0" SEQUENCE="false" COMMENT="Flag to show links or PlagLevel to students" PREVIOUS="show_report_to_students"/>
-                <FIELD NAME="ownerid" TYPE="char" LENGTH="255" NOTNULL="true" UNSIGNED="true" SEQUENCE="false" COMMENT="The id of the owner who created the submission" PREVIOUS="show_students_links" NEXT="submissionid"/>
-                <FIELD NAME="submissionid" TYPE="char" LENGTH="255" NOTNULL="false" DEFAULT="NULL" UNSIGNED="true" SEQUENCE="false" COMMENT="Unique PlagScan submission ID" PREVIOUS="ownerid" NEXT="enable_online_text"/>   
+                <FIELD NAME="ownerid" TYPE="char" LENGTH="255" NOTNULL="true" UNSIGNED="true" DEFAULT="0" SEQUENCE="false" COMMENT="The id of the owner who created the submission" PREVIOUS="show_students_links" NEXT="submissionid"/>
+                <FIELD NAME="submissionid" TYPE="char" LENGTH="255" NOTNULL="false" DEFAULT="ownerid" UNSIGNED="true" SEQUENCE="false" COMMENT="Unique PlagScan submission ID" PREVIOUS="ownerid" NEXT="enable_online_text"/>   
```

Comment:
Here I would delete the statmente 'DEFAULT="ownerid"' and directly do not put any default, so it will be 'null'.